### ISSUE TITLE
I cannot press the plus button on the right in the widgets on the main screen

### ISSUE THREAD

**Issue by devmdeniz** [8bitPit/Niagara-Issues#2390]
### Describe the bug

I cannot press the plus button on the right corner of the screen. A switch input can be entered for this. When the value of the input is "on", it may be the case that the alphabet does not come when the right side of the screen is pressed.

### Steps to reproduce

Add a widget to the home screen, preferably Google Tasks

### Does the issue also occur with other 3rd-party launchers?

Problem doesn't occur with other launchers

### Device name

Poco X3 Pro 256GB

### Android version

Android 13

### Screenshots / Screen recordings

https://github.com/8bitPit/Niagara-Issues/assets/68655812/1a58c169-7394-44a7-8679-3d68160bb0a2

**Comment by HDFLucky** [comment]
I believe this is the same issue I reported 3 weeks ago: https://github.com/8bitPit/Niagara-Issues/issues/2377

**Comment by devmdeniz** [comment]
> I believe this is the same issue I reported 3 weeks ago: #2377

I don't think this error is so difficult to resolve. I don't know if they are not trying or they don't care.
